pkg/httpclient/client.go

```diff
@@ -0,0 +1,28 @@
+package httpclient
+
+import (
+	"crypto/tls"
+	"net"
+	"net/http"
+	"time"
+)
+
+var (
+	dialer = &net.Dialer{
+		Timeout:   time.Second,
+		KeepAlive: 60 * time.Second,
+	}
+
+	transport = &http.Transport{
```

Comment:
can we allow the developer that use this SDK to pass its own `transport`?

Reply:
fixed